languages/js/src/types.ts

```diff
@@ -189,3 +189,9 @@ export interface QueryEvent {
 export type QueryResult = Generator<Map<string, any>, void, never>;
 
 export type obj = { [key: string]: any };
+
+export type EqualityFn = (x: any, y: any) => boolean;
+
+export interface OsoOptions {
```

Comment:
Can we please call this `Options`, or `ConstructorOptions`? Including `oso` in the name seems inconsistent. `PolarOptions` would be more accurate, but I understand wanting to avoid that.